README.md

```diff
@@ -127,6 +127,47 @@ docker cp ${CONTAINER}:/root/BUILD/rpms/x86_64/ ./rpms/
 docker rm ${CONTAINER}
 ```
 
+## Building a Hotfix
+
+Sometimes it is necessary to apply a patch without rebuilding the whole release in order to
+deliver a fix quickly.  This can be accomplished by building a hotfix rpm which will apply
+a series of patches.
+
+Create a patch file using `git format-patch` and copy to `rpm_spec/patches/`.  You will have to modify the path to be relative to the RPM BUILDDIR.
+
+For example if we wanted to create a hotfix to apply a commit in radjabov `1286aa34992340cad7a0778df01a845f1772e561` we would run:
+```sh
+git format-patch -1 1286aa34992340cad7a0778df01a845f1772e561
+```
+
+Which will create a patch file: `0001-Merge-pull-request-23123-from-agrare-fix_miq_request.patch`
+
+Make sure to remove any changes to spec files, as these are not packaged with the RPMs and will cause a conflict when
+the patch is applied.
+
+Next we have to change the file location to match the RPM BUILDDIR, for a patch in the core rpm this will be e.g. `manageiq-core-18.0-1`
+
+```
+diff --git a/manageiq-core-18.0-1/app/models/miq_provision_request_template.rb b/manageiq-core-18.0-1/app/models/miq_provision_request_template.rb
+index 621f20b7e7..f62f454ac5 100644
+--- a/manageiq-core-18.0-1/app/models/miq_provision_request_template.rb
++++ b/manageiq-core-18.0-1/app/models/miq_provision_request_template.rb
+```
+
+This ensures that the hotfix patch will apply correctly when it is run.  Then move that patch
+file to `rpm_spec/patches/` in the rpm_build repo.
+
+`docker build --pull --tag $USER/rpm_build:radjabov-hotfix .`
```

Comment:
FYI, I needed to supply the arch when building on my mac since we don't have aarch64 rpms in our manageiq rpm repository and my platform isn't the most common target arch: x86_64:

```
 podman build --pull --arch=x86_64 --tag $USER/rpm_build:radjabov-hotfix .
```

Reply:
That's probably an issue for all of the docker examples in this README.  Maybe we split out a PR to update all of them to work on a mac?

Reply:
oh for sure... I'm trying to validate it since I'm in this neck of the woods anyway.

Reply:
Is podman the way to go on mac silicon macs now?  Maybe we can drop the docker commands here now too

Reply:
Not sure.  That's what I'm using.  😉

Reply:
FYI, the next command also worked with podman:

```
podman run --rm -v `pwd`/OPTIONS:/root/OPTIONS -v `pwd`/BUILD:/root/BUILD $USER/rpm_build:radjabov-hotfix build_hotfix
```

Reply:
Yeah would be awesome if we could get one set of instructions that work for all, don't want to end up with another developer_setup with mac+ubuntu+centos yum+centos dnf for everything

Reply:
I'm using podman desktop, but docker is still "common"